pkg/kubelet/prober/prober_manager.go

```diff
@@ -166,12 +167,22 @@ func (t probeType) String() string {
 	}
 }
 
+func getSidecars(pod *v1.Pod) []v1.Container {
```

Comment:
I do not think we should call them sidecars in code, we should use `restartableInitContainer`